packages/xrpl/test/models/NFTokenCreateOffer.test.ts

```diff
@@ -16,7 +16,7 @@ describe('NFTokenCreateOffer', function () {
       TransactionType: 'NFTokenCreateOffer',
       NFTokenID: NFTOKEN_ID,
       Amount: '1',
-      Owner: 'r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ',
+      Owner: 'rcXY84C4g14iFp6taFXjjQGVeHqSCh9RX',
```

Comment:
Why were all of these changed?

Reply:
The previous value was an invalid classic address.  Looking at it now I could have just deleted a character.

Reply:
That didnt work because it is still not valid.  The examples lower were too short.